packages/manager/apps/container/src/identity-documents-modal/IdentityDocumentsModal.tsx

```diff
@@ -0,0 +1,137 @@
+import { kycIndiaModalLocalStorageKey, kycIndiaFeature, requiredStatusKey } from "./constants";
+import { useIdentityDocumentsStatuts } from "@/hooks/useIdentityDocumentsStatuts";
+import { ODS_BUTTON_SIZE, ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components';
+import { FunctionComponent, useEffect, useRef, useState } from "react";
+import { useFeatureAvailability } from "@ovhcloud/manager-components";
+import { useTranslation, Trans } from 'react-i18next';
+import { useLocalStorage } from "react-use";
+import { useShell } from "@/context";
+import {
+    OsdsButton,
+    OsdsCollapsible,
+    OsdsModal,
+    OsdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+    ODS_THEME_COLOR_HUE,
+    ODS_THEME_COLOR_INTENT,
+    ODS_THEME_TYPOGRAPHY_LEVEL,
+    ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+
+export const IdentityDocumentsModal: FunctionComponent = () => {
+    const shell = useShell();
+    const navigationPlugin = shell.getPlugin('navigation');
+    const [storage, setStorage] = useLocalStorage<boolean>(kycIndiaModalLocalStorageKey);
+
+    const { t } = useTranslation('identity-documents-modal');
+    const legalInformationRef = useRef<any>(null);
+
+    const [showModal, setShowModal] = useState<boolean>(false);
+
+    let availabilityDataResponse;
+    if (!storage) {
+        availabilityDataResponse = useFeatureAvailability([kycIndiaFeature]);
+    }
+    
+    const availability = availabilityDataResponse?.data;
```

Comment:
**nitpick:** This can be simplified.
```suggestion
    let kycIndiaAvailability;
    if (!storage) {
        kycIndiaAvailability = useFeatureAvailability([kycIndiaFeature])?.data;
    }
```